component/input/input.css

```diff
@@ -0,0 +1,33 @@
+/* General */
+
+a {
+    text-decoration: none;
+}
+
+/* Typography */
+
+:root {
+    --font-family: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue",
+    Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji",
+    "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+}
+
+/* Input */
+
+.inp {
+    padding: 10px;
+    border-radius: 10px;
+    border: 3px solid #17a2b8;
+}
+
+.inp-error {
+    border: 3px solid #dc3545;
+}
+
+.inp-warning {
+    border: 3px solid #ffc107;
+}
+
+.inp-success {
+    border: 3px solid #28a745;
```

Comment:
Using variables for these colours is one best practice as these colour validations are crucial for the input component.

Reply:
got it